dcos/util.py

```diff
@@ -726,3 +726,19 @@ def read_file_json(path):
         path = os.path.expanduser(path)
         with open_file(path) as options_file:
             return load_json(options_file)
+
+
+def get_fault_domain(state):
+    """ Read mesos state json and return region and zone
+
+    :param state: mesos state json top level or a node item
+    :type state: dict
+    :returns: region and zone
+    :rtype: str | None
```

Comment:
As this returns a tuple and not a single value, should this be `:rtype: (str | None, str | None)`?

Reply:
fixed